samples/dymaptic.GeoBlazor.Core.Sample.Shared/Shared/NavMenu.razor

```diff
@@ -87,7 +87,8 @@
         new("projection-tool", "Projection Tool", "oi-cog"),
         new("basemap-projections", "Basemap Projections", "oi-flash"),
         new("unique-value", "Unique Renderers", "oi-brush"),
-    new("marker-rotation", "Marker Rotation", "oi-loop-circular")
+        new("marker-rotation", "Marker Rotation", "oi-loop-circular"),
+        new("/geometry-selection", "Geometry Selection", "oi-compass"),
```

Comment:
can you find a different openiconic icon for this new page? We're already using `oi-compass` for the `Navigation` page.